app/src/main/res/drawable/bg_blue_card.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android"
+       android:padding="16dp"
+       android:shape="rectangle">
+  <stroke
+    android:width="1.5dp"
```

Comment:
`1.5 dp` why? I suggest not using such values,  probably we should be them in multiple of `4` or sometimes `2` if required.

Reply:
It looks too thick if I update to 2.